client/state/themes/actions/theme-transfer.js

```diff
@@ -12,7 +12,7 @@ import {
 
 import 'calypso/state/themes/init';
 
-function initiateTransfer( siteId, plugin, theme, onProgress ) {
+function initiateTransfer( siteId, plugin, theme, geoAffinity, onProgress ) {
```

Comment:
🌶️ Living dangerously and adding new params in the middle of the param list! 😥 

I did a search and it looks like you've caught all the cases thought 👍

Reply:
The function isn't exported, fortunately!